pkg/vsphere/disk/disk_manager_test.go

```diff
@@ -19,21 +19,41 @@ import (
 	"os"
 	"strings"
 	"testing"
+	"time"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/pkg/vsphere/session"
 	"github.com/vmware/vic/pkg/vsphere/tasks"
-	"github.com/vmware/vic/pkg/vsphere/test"
+	"github.com/vmware/vic/pkg/vsphere/test/env"
 	"golang.org/x/net/context"
 )
 
+func Session(ctx context.Context, t *testing.T) *session.Session {
+	config := &session.Config{
+		Service: env.URL(t),
+
+		DatastorePath: env.DS(t),
+
+		Insecure:  true,
+		Keepalive: time.Duration(5) * time.Minute,
+	}
+
+	s, err := session.NewSession(config).Create(ctx)
+	if err != nil || s.IsVSAN(ctx) {
```

Comment:
This should have a big fat comment as to why we're skipping this test if it's VSAN.  Will add.  The TL;DR is it's too complicated to make it work on vsan without the `datastore` struct doing all of the URI mangling.  And importing that package leads to a circular dependency.